GiniSwitchSDK/GiniSwitchSDK/Classes/Error.swift

```diff
@@ -15,6 +15,7 @@ public enum GiniSwitchErrorCode: Int {
     case network
     case authentication
     case tokenExpired
+    case grantInvalid
```

Comment:
No ;(. The idea was to have an "errorCode" property in the response JSON. That, in turn would set this value. However, that's still not implemented and we have to rely on the "error" property, which is a string. So, now we check for `error = "invalid_grant"`. Same goes for "invalid_token". What do you think, should we give up and remove the whole enum? Or hope we can use it in the future?

Reply:
If it is not used I would remove it, because it's just confusing for now. If it will be implemented in the future it's added pretty fast and have to be tested anyway.